provider/kubernetes.go

```diff
@@ -196,10 +199,10 @@ func (provider *Kubernetes) loadIngresses(k8sClient k8s.Client) (*types.Configur
 						endpoints, exists, err := k8sClient.GetEndpoints(service.ObjectMeta.Namespace, service.ObjectMeta.Name)
 						if err != nil || !exists {
```

Comment:
A bad rebase is why actually. Specifically wrote some test code for it. Will fix,